src/js/components/BreadcrumbComponent.jsx

```diff
@@ -0,0 +1,87 @@
+var React = require("react/addons");
+var Link = require("react-router").Link;
+
+var ViewHelper = require("../helpers/ViewHelper");
+
+var BreadcrumbComponent = React.createClass({
+  displayName: "BreadcrumbComponent",
+
+  propTypes: {
+    app: React.PropTypes.string,
+    group: React.PropTypes.string,
+    task: React.PropTypes.string
+  },
+
+  getGroupLinks: function () {
+    var group = this.props.group;
+    if (group == null) {
+      return null;
+    }
+
+    var pathParts = group.split("/").slice(0, -1);
```

Comment:
Surprisingly even if using a usual slice instead of the sequence in the .map() is much slower..